react-ui/src/aboutus/aboutus.js

```diff
@@ -0,0 +1,83 @@
+                                <Accordion.Item eventKey="0">
+                                    <ListGroup defaultActiveKey={['0']} alwaysOpen>
+                                        <ListGroup.Item  as={Link} to="/metricwiki" state={{token: auth}}>Metric Wiki</ListGroup.Item>
+                                    </ListGroup>
+                                    <Accordion.Body style={{ paddingTop: "5px" }}>
+                                        <Nav.Link style={{ borderBottom: "1px solid #750E21" }} as={Link} to="/metricwiki" state={{token: auth}}>Lead Time</Nav.Link>
+                                        <Nav.Link style={{ borderBottom: "1px solid #750E21" }} as={Link} to="/metricwiki" state={{token: auth}}>Cycle Time</Nav.Link>
+                                        <Nav.Link style={{ borderBottom: "1px solid #750E21" }} as={Link} to="/metricwiki" state={{token: auth}}>Burndown Chart</Nav.Link>
+                                        <Nav.Link style={{ borderBottom: "1px solid #750E21" }} as={Link} to="/metricwiki" state={{token: auth}}>Dev Focus</Nav.Link>
+                                        <Nav.Link style={{ borderBottom: "1px solid #750E21" }} as={Link} to="/metricwiki" state={{token: auth}}>Cruft</Nav.Link>
+                                    </Accordion.Body>
+                                </Accordion.Item>
+                            </Accordion>
+                            <ListGroup defaultActiveKey={['0']} alwaysOpen>
+                                <ListGroup.Item as={Link} to="/aboutus" state={{token: auth}}><b>About Us</b></ListGroup.Item>
+                            </ListGroup>
+                        </Nav>
+                    </div>
+                </div>
+                <div style={{ marginLeft: "25vh" }}>
+                <div className="backgroundLightOrange" style={{ minHeight: "10vh", borderBottom: "1px solid #750E21" }}>
+                        <Navbar>
+                            <div style={{ marginLeft: "40px", marginTop: "10px", fontFamily: "Cascadia Mono SemiLight"}}><h2><b>DASHBOARD</b></h2></div>
+                            <div className="ms-auto" style={{ marginRight: "45px", marginTop: "5px" }}>
+                                <a href="/" style={{ fontSize: "20px" }}><u>Logout</u></a>
+                            </div>
+                        </Navbar>
+                    </div>
+                    <div className="backgroundWhite" style={{ minHeight: "85vh", padding: "30px" }}>
+                        <Container>
+                            <Row>
+                                <Col sm={8}>
+                                    <h2>Meet the Developer Team!</h2><br />
+                                    We are a developer team of five members: Ashutosh Chaurasia (achaur11@asu.edu), Advait Chirmule (achirmul@asu.edu), Hasan Shahid (hshahid@asu.edu),
+                                    Sehun Park (spark29@asu.edu) and Shikha Verma (sverma27@asu.edu).
```

Comment:
Change misspelling (spark29 to spark290)

Reply:
Fixed